web_fragments/test_utils/views.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Simple fragment view for testing.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from web_fragments.fragment import Fragment
+from web_fragments.test_utils import TEST_CSS, TEST_HTML, TEST_JS
+from web_fragments.views import FragmentView
+
+TEST_FRAGMENT_VIEW_NAME = 'test_fragment_view'
+
+
+class TestFragmentView(FragmentView):
+    """
+    Simple fragment view for testing.
+    """
+
+    def render_fragment(self, request, **kwargs):
+        """
+        Returns a simple fragment
+        """
+        fragment = Fragment(TEST_HTML)
+        fragment.add_javascript(TEST_JS)
+        fragment.add_css(TEST_CSS)
+        return fragment
```

Comment:
Organizationally, this is hard to find. One of the core purposes of tests is to document how the library is used. This is an excellent example of a basic Django fragment view, and gives a clear idea of what the API is and how to use it. I

It's hidden away under `test_utils`. 

If you're looking by naming/organization, you'll instead find, `web_fragments/tests/test_views.py`. That is a very opaque file. Unless you're already familiar with how fragments work, and pretty advanced Django testing, you won't understand it. At a quick glance, it also doesn't guide you here. You don't know this is what it's testing. 

The only reason I knew this existed is because I read the entire code.